src/kmeans.jl

```diff
@@ -44,20 +45,34 @@ function kmeans{T<:AbstractFloat}(X::Matrix{T}, k::Int;
                 weights=nothing,
                 init=_kmeans_default_init,
                 maxiter::Integer=_kmeans_default_maxiter,
+                n_init::Integer=_kmeans_default_n_init,
```

Comment:
I understand that `n_init` comes from Python's sklearn (#64), but it doesn't sound like a best choice for me.
Maybe something like `n_tries` to reflect that the parameter defines how many times the algorithm, rather than some initialization procedure, is run?

Reply:
or `ntries` ?

Reply:
10 is what sklearn does at it sounds reasonable to me.
It isn't unusual to run 1000s of times, (that was done as the baseline for the affinity propagation paper)
If some need a quick partition they can ask for it.

The default shouldn't be so sensitive to random factors.

I think 10 strikes the right balance.
Though I could see argument for 3 or 30